exporters/committime/app.py

```diff
@@ -111,11 +118,14 @@ def generate_ld_metrics_list(namespaces):
         # only use builds that have the app label
         builds = v1_builds.get(namespace=namespace, label_selector=loader.get_app_label())
 
+
         # use a jsonpath expression to find all values for the app label
-        jsonpath_str = 'items[*].metadata.labels.%s' % (loader.get_app_label())
+        jsonpath_str = "$['items'][*]['metadata']['labels'][" + str(loader.get_app_label()) + "]"
```

Comment:
i think this line needs to change to:

```
jsonpath_str = "$['items'][*]['metadata']['labels']["' + str(loader.get_app_label()) + "']"
```

Reply:
This produces errors in my IDE, it doesn't seem like valid python syntax

Reply:
oh, I had one of my quotes misplaced:

```
jsonpath_str = "$['items'][*]['metadata']['labels']['" + str(loader.get_app_label()) + "']"
```